content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+   report, then :menuselection:`Toggle Studio --> Reports --> Select or Create a report --> Report
+   --> Select a Paper format`.
+
+.. image:: pdf_reports/default-layout.png
+   :align: center
+   :alt: Configuration pop-up window for the default layout of PDF reports
+
+.. _studio/pdf-reports/header-footer:
+
+Header and footer
+=================
+
+When creating a new report in Studio, you must choose between one of three styles of reports first.
+This is solely used to determine what is displayed on the header and footer.
+
+.. _studio/pdf-reports/header-footer/external:
+
+External
+--------
+
+The header displays the company :ref:`studio/pdf-reports/default-layout/logo` and several values
+set on the *Company* model: the :guilabel:`Company Name`, :guilabel:`Phone`, :guilabel:`Email`, and
+:guilabel:`Website`.
+
+.. tip::
+   To change a company's information, go to :menuselection:`Settings --> Companies --> Update Info`.
+
+.. image:: pdf_reports/external-header.png
+   :align: center
+   :alt: Example of an External header
+
+The footer displays the values set on the :ref:`studio/pdf-reports/default-layout/footer`,
+:ref:`studio/pdf-reports/default-layout/details`, and
+:ref:`studio/pdf-reports/default-layout/tagline` fields, as well as the page number.
+
+.. image:: pdf_reports/external-footer.png
+   :align: center
+   :alt: Example of an External footer
+
+.. _studio/pdf-reports/header-footer/internal:
+
+Internal
+--------
+
+The header displays the user's current date and time, :guilabel:`Company Name`, and page number.
+
+There is no footer.
+
+.. _studio/pdf-reports/header-footer/blank:
+
+Blank
+-----
+
+There is neither a header nor a footer.
+
+.. _studio/pdf-reports/elements:
+
+Add tab
+=======
+
+After opening an existing report or creating a new one, go to the :guilabel:`Add` tab to add or edit
+elements. The elements are organized into four categories: :ref:`studio/pdf-reports/elements/block`,
+:ref:`studio/pdf-reports/elements/inline`, :ref:`studio/pdf-reports/elements/table`, and
+:ref:`studio/pdf-reports/elements/column`.
+
+.. _studio/pdf-reports/elements/block:
+
+Block
+-----
+
+Block elements start on a new line and occupy the full width of the page.
+
+.. tip::
+   You can set an element's width by selecting it and going to the :guilabel:`Options` tab if you do
+   not want it to use the full width.
+
+- :guilabel:`Text`: add any text using small font size by default.
+
+- :guilabel:`Title Block`: add any text using larger font size by default.
+
+- :guilabel:`Image`: add an image. You can either upload one from your device, add one from
+  an URL, or select one already existing on your database.
+
+- :guilabel:`Field`: dynamically add a field's value.
+
+- :guilabel:`Field & Label`: to dynamically add a field's value and label.
+
+- :guilabel:`Address Block`: to dynamically add the values, if any, of a contact's: *Name*,
```

Comment:
important: specifiy that the field chosen for that element must be a Contact (res.partner)